src/css/App.css

```diff
@@ -26,3 +26,42 @@
 	width: 50%;
 	margin: auto;
 }
+
+.pageSliderLeft-enter {
+  transform: translate3d(100%, 0, 0);
+  /* background:rgba(0,0,0,1);
+  display:none;
+  width:100%; 
+  height:100%;
+  position: relative;; 
+  top:0; 
+  left:0; 
+  z-index:99998; */
+}
+.pageSliderLeft-enter.pageSliderLeft-enter-active {
+  transform: translate3d(0, 0, 0);
+  transition: all 600ms;
+  
+}
+.pageSliderLeft-exit {
+  transform: translate3d(0, 0, 0);
+}
+.pageSliderLeft-exit.pageSliderLeft-exit-active {
+  transform: translate3d(100%, 0, 0);
+  transition: all 600ms;
+}
+
+.pageSliderRight-enter {
+  transform: translate3d(-100%, 0, 0);
+}
+.pageSliderRight-enter.pageSliderRight-enter-active {
+  transform: translate3d(0, 0, 0);
+  transition: all 600ms;
+}
+.pageSliderRight-exit {
+  transform: translate3d(0, 0, 0);
+}
+.pageSliderRight-exit.pageSliderRight-exit-active {
+  transform: translate3d(-100%, 0, 0);
+  transition: all 600ms;
+}
```

Comment:
I'm assuming this is css for the page transitions? If so, then this should be removed since we aren't using them anymore.